resources/clj-kondo.exports/tensegritics/clojuredart/config.edn

```diff
@@ -1,3 +1,6 @@
 {:linters {:non-arg-vec-return-type-hint {:level :off}
-           :flutter/widget {:level :error}} 
+           :flutter/widget {:level :error}
+           :clojure-lsp/unused-public-var {:exclude #{main}}
+           :unresolved-namespace {:exclude [dart dart:core]}
+           :unresolved-symbol {:exclude [bool Function fallback List Map]}} 
```

Comment:
I am not sure about fallback here – will users be using this in their code? if not, I will move it back to /root/.clj-kondo.

Reply:
Yes they can use fallback